lua/telescope/pickers.lua

```diff
@@ -414,13 +414,15 @@ function Picker:find()
     pcall(a.nvim_buf_set_option, prompt_bufnr, "filetype", "TelescopePrompt")
     pcall(a.nvim_buf_set_option, results_bufnr, "filetype", "TelescopeResults")
 
-    -- TODO(async): I wonder if this should actually happen _before_ we nvim_buf_attach.
-    -- This way the buffer would always start with what we think it should when we start the loop.
```

Comment:
why are you removing todos ?

Reply:
I figured that this PR adds the async part, just in a different way, while the suggested way wouldn't work anymore.

Reply:
not really but its no longer possible due to prompt buffer changes anyway. So its leave it removed